src/ring_theory/fractional_ideal.lean

```diff
@@ -810,12 +811,18 @@ begin
     have y_zero := (mul_eq_zero.mp this).resolve_left
       (mt ((algebra_map R₁ K).injective_iff.1 (is_fraction_ring.injective _ _) _)
           (mem_non_zero_divisors_iff_ne_zero.mp haJ)),
-    exact not_mem_zero ((mem_zero_iff R₁⁰).mpr y_zero) },
+    apply not_mem_zero,
+    simpa only using (mem_zero_iff R₁⁰).mpr y_zero, },
   intros b hb,
   convert hI _ (hb _ (submodule.smul_mem _ aJ mem_J)) using 1,
   rw [← hy', mul_comm b, ← algebra.smul_def, mul_smul]
 end
 
+lemma fractional_div_of_nonzero {I J : fractional_ideal R₁⁰ K} (h : J ≠ 0) :
+  is_fractional R₁⁰ (I / J : submodule R₁ K) :=
+I.is_fractional.div_of_nonzero J.is_fractional $ λ H, h $
+  coe_to_submodule_injective $ H.trans coe_zero.symm
```

Comment:
This was kind of annoying because `coe_zero` is not true by `rfl`, which is surprising.

Reply:
I don't think we can make the following two equalities both defeq:
 * `(((⊥ : ideal R) : fractional_ideal R A) : set A) = (⊥ : fractional_ideal R A) : set A) = ((⊥ : submodule R A) : set A)`

At least, if we keep the definitions as they are:
 * `(((I : ideal R) : fractional_ideal R A) : set A) = algebra_map R A '' (I : set R)`
 * `((⊥ : ideal R : set R) = {0}`
 * `((⊥ : submodule R A) : set A) = {0}`

Substituting gives the following problem, which we cannot make defeq:
 * `{algebra_map R A 0} = {0}`